css/styles.css

```diff
@@ -2,9 +2,34 @@
   font-family: 'Noto Sans JP', sans-serif;
 }
 
-header {
-  background-color: rgb(75, 75, 75);
-  height: 60px;
+.navbar {
+  background-color: #e4635b;
+  color: white;
+  height: 65px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: 0;
+  list-style: none;
+}
+
+.nav-link {
+  padding-right: 4rem;
+  font-size: 18px;
+}
+
+.main-logo {
+  font-size: 20px;
+  font-weight: bold;
+  letter-spacing: 0.05rem;
+}
+
+.inactive {
+  display: none;
+}
+
+.active {
+  display: block;
 }
```

Comment:
Rather than having this class, you can just remove `hidden` when you want an element to be visible again